apps/src/webapp/extraControlRows.html.ejs

```diff
@@ -19,6 +19,12 @@
   </div>
 <% } %>
 
+<div id="view-data-cell" class="view-data-cell-none">
```

Comment:
I think this should be inside the if (debugButtons) or a similar conditional. I think when config.hide_source is true, we don't want to show just this button.